app/User.php

```diff
@@ -70,12 +71,12 @@ public function setName($name)
 
     public function getLastName()
     {
-        return $this->attributes['lastName'];
+        return $this->attributes['last_name'];
     }
 
-    public function setLastName($lastName)
+    public function setLastName($last_name)
```

Comment:
The fields of the tables use snake_case, but here you should use the camel case notation as you had before: lastName